jsonnet/honeycomb-agent-ds-base.libsonnet

```diff
@@ -1,4 +1,5 @@
-local k = import "/Users/alex/src/go/src/github.com/ksonnet/ksonnet-lib/ksonnet.beta.2/k.libsonnet";
+// CHANGE THIS IMPORT TO POINT TO YOUR LOCAL KSONNET
+local k = import "/Users/jyao/heptio/hausdorff-ksonnet/ksonnet.beta.2/k.libsonnet";
```

Comment:
You can use the `-J` flag in the Jsonnet CLI to have this pick up the library from an arbitrary path.